<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interfaz</class>
 <widget class="QWidget" name="Interfaz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>633</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">BACKGROUND-COLOR:rgb(255, 254, 212)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="19" column="8">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pbSalir">
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="8">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="8">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(107, 3, 115);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Verdana</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="7">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="8" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lTotalCant">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Verdana</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(56, 106, 4);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="1">
      <widget class="QCheckBox" name="chkRecup">
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Recuperados</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkConfir">
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Confirmados</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="chkMuert">
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Muertes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="19" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_8"/>
   </item>
   <item row="0" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QComboBox" name="listPaises">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="listRegiones">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>163</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pbCalcular">
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string>Calcular Gr√°ficas</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="4">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="8">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QLabel" name="lUltFecha">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="lPriFecha">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="8">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbSalir</sender>
   <signal>released()</signal>
   <receiver>Interfaz</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listPaises</sender>
   <signal>highlighted(QString)</signal>
   <receiver>Interfaz</receiver>
   <slot>listar_regiones(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCalcular</sender>
   <signal>pressed()</signal>
   <receiver>Interfaz</receiver>
   <slot>calcular_datos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCalcular</sender>
   <signal>pressed()</signal>
   <receiver>Interfaz</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkConfir</sender>
   <signal>clicked()</signal>
   <receiver>Interfaz</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRecup</sender>
   <signal>clicked()</signal>
   <receiver>Interfaz</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMuert</sender>
   <signal>clicked()</signal>
   <receiver>Interfaz</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>sTexto(QString)</signal>
  <signal>sRango(int,int)</signal>
  <signal>sValor(int)</signal>
  <slot>listar_regiones(QString)</slot>
  <slot>calcular_datos()</slot>
  <slot>actualizar_bd()</slot>
  <slot>proximamente()</slot>
  <slot>mostrar_SIR()</slot>
 </slots>
</ui>
